databricks/koala/structures.py

```diff
@@ -560,8 +560,10 @@ def get(self, key, default=None):
         except (KeyError, ValueError, IndexError):
             return default
 
-    def sort_values(self, by):
-        df = self._spark_sort(by)
+    def sort_values(self, by, ascending=True):
```

Comment:
Could you add a doc?

Reply:
Also adding `inplace` here would be great, otherwise could you add a TODO comment to support `inplace` later?

Reply:
I guess we can support `na_position` as well?

Reply:
Regarding `na_position` - is there any easy way for me to add support for that ?
https://spark.apache.org/docs/2.4.0/api/python/pyspark.sql.html#pyspark.sql.DataFrame.sort
Spark doesn't have any direct equivalent for it so, handling all cases seemed a bit complicated

Reply:
Actually I didn't try it, but I was thinking of using `Column.asc_nulls_first`, `.asc_nulls_last`, `.desc_nulls_first`, and `.desc_nulls_last`.
You can leave it. Thanks.

Reply:
Had forgotten about those as I use Spark 2.3 as my default spark :D
Implemented it using those column-functions